libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.m

```diff
@@ -392,20 +396,48 @@ - (BOOL)deleteAccountForUser:(SFUserAccount *)user error:(NSError **)error {
 }
 
 - (SFUserAccount *)applyCredentials:(SFOAuthCredentials*)credentials {
+    return [self applyCredentials:credentials withIdData:nil];
+}
+
+- (SFUserAccount *)applyCredentials:(SFOAuthCredentials*)credentials withIdData:(SFIdentityData *) identityData {
+    return [self applyCredentials:credentials withIdData:identityData andNotification:YES];
+}
 
-    SFUserAccountChange change = SFUserAccountChangeCredentials;
+- (SFUserAccount *)applyCredentials:(SFOAuthCredentials*)credentials withIdData:(SFIdentityData *) identityData andNotification:(BOOL) shouldSendNotification{
```

Comment:
Okay.  I'd argue that there's a general refactor point for isolating account creation from account notification, but it works this way too.